### PR TITLE
Update schedule.md

### PR DESCRIPTION
**Pull request by creisle**

### CODE REVIEW

**Review by kmnip — COMMENTED** [review]
Looks good! Please see my suggested changes.

File: src/schedule.md [hunk 1]
```diff
@@ -1,106 +1,49 @@
...
-## Fall 2021
+## Fall 2022
 
 <div class="timeline" markdown="1">
 
-- September 22nd, 2021, Wednesday
+- September 15th, 2022, Thursday
 
-    :material-clock: 11:00am PST
+    :material-clock: 6:00pm - 9:00pm PST
```
[the remark this answers is not in the document]
Comment (cmrn-rhi) [reply src/schedule.md:9]: True, it's PDT until the Nov session. We could alternatively just use PT (Pacific Time) if we don't want to dance around these.

File: src/schedule.md [hunk 2]
```diff
@@ -1,106 +1,49 @@
...
 
-    :material-map-marker: online
+    :material-map-marker:  St. Paul's Hospital Site: [Cullen Room (SPH1477)](https://drive.google.com/file/d/1VBdWxJj_WJeWCcdox4RY5wFWI2F5Gfgy/view?usp=sharing)
 
-    Speaker: **Adi Steif**
+    Speaker: **TBA**
 
-    Short presentation: **Junbum Im (Kuchenbauer lab, BCCRC)**
+    Short presentation: **TBA**
```
[the remark this answers is not in the document]
Comment (creisle) [reply src/schedule.md:25]: FYI, you can just comment these once, I'll use a find and replace generally to fix them so I'll find the others :)

File: src/schedule.md [hunk 3]
```diff
@@ -1,106 +1,49 @@
...
-- [April 20th, 2022, Wednesday](./archive/2022/2022-04-20.md)
-
-    :material-clock: 11:00am PST online
-
-    :material-map-marker:  online
-
-    Speaker: **Carl de Boer**
-
-    Short presentation: **Bradley R. Jones (Joy lab, UBC)**
+    Short presentation: **TBA**
```
[the remark this answers is not in the document]
Comment (creisle) [reply src/schedule.md]: https://github.com/van-bug/website/pull/5/commits/f2142b5800c86b3071e7f8e66f7efba0540a0bdc

File: src/schedule.md [hunk 4]
```diff
@@ -1,106 +1,49 @@
...
+    :material-clock: 6:00pm - 9:00pm PST
 
-    :material-map-marker:  online
+    :material-map-marker:  St. Paul's Hospital Site: [Cullen Room (SPH1477)](https://drive.google.com/file/d/1VBdWxJj_WJeWCcdox4RY5wFWI2F5Gfgy/view?usp=sharing)
 
-    Speaker: **Jesse Gillis**
+    Speaker: **TBA**
 
-    Short presentation: **Dalia Alghamdi (Hsiao lab, SFU)**
+    Short presentation: **TBA**
```
[the remark this answers is not in the document]
Comment (creisle) [reply src/schedule.md]: https://github.com/van-bug/website/pull/5/commits/f2142b5800c86b3071e7f8e66f7efba0540a0bdc

File: src/schedule.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (creisle) [reply src/schedule.md]: https://github.com/van-bug/website/pull/5/commits/f2142b5800c86b3071e7f8e66f7efba0540a0bdc

File: src/schedule.md [hunk 5]
```diff
@@ -1,106 +1,49 @@
...
+    :material-clock: 6:00pm - 9:00pm PST
 
-    :material-map-marker:  online
+    :material-map-marker:  St. Paul's Hospital Site: [Cullen Room (SPH1477)](https://drive.google.com/file/d/1VBdWxJj_WJeWCcdox4RY5wFWI2F5Gfgy/view?usp=sharing)
 
-    Speaker: **Tao Huan**
+    Speaker: **Geoffrey Schiebinger**
 
-    Short presentation: **Shreya Bendre (Bénard/Lin lab, BCCRC)**
+    Short presentation: **TBA**
```
[the remark this answers is not in the document]
Comment (creisle) [reply src/schedule.md]: https://github.com/van-bug/website/pull/5/commits/f2142b5800c86b3071e7f8e66f7efba0540a0bdc

File: src/schedule.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (creisle) [reply src/schedule.md]: https://github.com/van-bug/website/pull/5/commits/93388cae7c7b57fba18980b6390837d78f6121e5

File: src/schedule.md [hunk 6]
```diff
@@ -1,106 +1,49 @@
...
 
-- October 20th, 2021, Wednesday
+- October 20th, 2022, Thursday
 
-    :material-clock: 11:00am PST online
+    :material-clock: 6:00pm - 9:00pm PST
```
[the remark this answers is not in the document]
Comment (creisle) [reply src/schedule.md]: https://github.com/van-bug/website/pull/5/commits/93388cae7c7b57fba18980b6390837d78f6121e5

**Review by kmnip — APPROVED** [review]
Thanks for the changes!

**Pull request merged**